bin/bin/dune

```diff
@@ -1,7 +1,7 @@
 (executables
  (package      rwo)
- (names        app       dep)
- (public_names rwo-build rwo-dep)
+ (names        app       dep       output)
```

Comment:
I think we should give up on a trying to align names and public names, especially since they aren't here!

As a matter of fact we should even let dune format our dune files.

Reply:
We should also try to come up with a better name than `output` as it's a bit vague, especially now that it's a standalone executable and not a subcommand of `mdx`.

What do you think about `convert_md`/`rwo-convert-md`?

Reply:
Ho, so that's why those spaces were there, I could not figure it, I'll get rid of that.

For the name, rwo-convert_md looks good to me